test/python/circuit/test_extensions_standard.py

```diff
@@ -1373,6 +1373,8 @@ def test_to_matrix(self):
                     gate = gate_class("IXYZ")
                 elif gate_class == BooleanExpression:
                     gate = gate_class("x")
+                elif gate_class.__name__ == 'Isometry':
```

Comment:
Was it possible to check against the class object itself here as above?

Reply:
Yes, I reverted isometry to an instruction in be42d08.